python/lsst/pipe/tasks/calexpCutout.py

```diff
@@ -104,10 +104,11 @@ def run(self, in_table, calexp):
             pt = geom.SpherePoint(geom.Angle(ra, geom.degrees),
                                   geom.Angle(dec, geom.degrees))
             pix = wcs.skyToPixel(pt)
-            size = rec['size'].value
+            width = rec['xspan'].value
+            height = rec['yspan'].value
```

Comment:
Please consider using `xspan` and `yspan` for variable names, in order to make it more obvious where they come from.

I can see arguments both ways. If you prefer what you have then by all means keep it.